packages/katello/katello-repos/katello-repos.spec

```diff
@@ -95,6 +95,10 @@ for repofile in %{buildroot}%{repo_dir}/*.repo; do
     sed -i "s/@PULP_VERSION@/%pulp_version/" $repofile
 done
 
+%if 0%{?rhel} == 5
+  sed -i "s/gpgcheck=1/gpgcheck=0/" %{buildroot}%{repo_dir}/katello-client.repo
```

Comment:
`sed -i '/gpgcheck/ s/1/0/'` is probably more robust (as it will also work for lines like `gpgcheck =1` or `gpgcheck = 1`, but not strictly required as the file we ship does contain `gpgcheck=1` and that's what we modify.

Reply:
I like it because it'll catch any situations where someone may have edited the file by hand. updated!

Reply:
We're running that during `%build`, so it's always our unedited version :)

Reply:
:facepalm: haha